Formula/slowhttptest.rb

```diff
@@ -13,15 +14,23 @@ class Slowhttptest < Formula
     sha256 "c9b36ccf8aee0f6572e2eb1112caf25811c22f1500ad81c1277309c76bd6460b" => :high_sierra
   end
 
+  # Remove these in next version
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
   depends_on "openssl@1.1"
 
   def install
+    system "sed", "-i", "''", "s/1\.8\.1/1\.8\.2/", "configure.ac" # Remove in next version
+    system "autoconf" # Remove in next version
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
     system "make", "install"
   end
 
   test do
     system "#{bin}/slowhttptest", "-u", "https://google.com",
                                   "-p", "1", "-r", "1", "-l", "1", "-i", "1"
+    # Remove in next version
+    assert_match "1\.8\.2", shell_output("#{bin}/slowhttptest -h", 1)
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/slowhttptest -h", 1)
```

Reply:
Good idea. The author has a history of forgetting to bump the version in new releases.